src/govuk/all.mjs

```diff
@@ -56,7 +56,7 @@ function initAll (config) {
 
   // Find first error summary module to enhance.
   var $errorSummary = $scope.querySelector('[data-module="govuk-error-summary"]')
-  new ErrorSummary($errorSummary).init()
+  new ErrorSummary($errorSummary, config.errorSummary).init()
```

Comment:
Can you add `config.errorSummary` to the initAll doc block?

(I'm still not 100% sure this is going to scale very well, but let's be consistent)